indico/util/mdx_latex_test.py

```diff
@@ -0,0 +1,29 @@
+__author__ = 'mic4ael'
+
+from indico.util import mdx_latex
+from markdown import Markdown
+import pytest
+
+
+_md = Markdown(safe_mode='remove')
+_latex_md = mdx_latex.LaTeXExtension()
+_latex_md.extendMarkdown(_md, _md.__dict__)
+
+
+@pytest.mark.parametrize("input,expected", [
+    ('\nplain text\n', 'plain text'),
+    ('plain text with reserved characters &^{}', r'plain text with reserved characters \&\^{}\{\}'),
+    ('**bold text**', r'\textbf{bold text}'),
+    ('**bold with &reserved{} characters**', r'\textbf{bold with \&reserved\{\} characters}'),
+    ('*italic text*', r'\emph{italic text}'),
+    ('*italic text with reserved chars &{}*', r'\emph{italic text with reserved chars \&\{\}}'),
+    ('[link with reserved chars&{}](www.indico.github.io)', r'\href{www.indico.github.io}{link with reserved chars\&\{\}}'),
+    ('> Plain Blockquote', '\\begin{quotation}\nPlain Blockquote\n\\end{quotation}'),
+    ('> Blockquote with reserved chars &{}', '\\begin{quotation}\nBlockquote with reserved chars \&\{\}\n\\end{quotation}'),
+    ('\tPlain code sample', '\\begin{verbatim}\nPlain code sample\n\\end{verbatim}'),
+    ('\tCode sample with reserved chars &{}', '\\begin{verbatim}\nCode sample with reserved chars &{}\n\\end{verbatim}'),
+    ('## Plain Heading ##', '\subsection{Plain Heading}'),
+    ('## Heading with reserved chars &{} ##', '\subsection{Heading with reserved chars \&\{\}}'),
+])
+def test_convert_markdown_to_latex(input, expected):
+    assert(_md.convert(input) == expected)
```

Comment:
Could try wrapping it between the two tuple elements... but if it's still too long I'm not sure if it makes lots of sense to try wrapping it. It's  just test data after all...